src/skyscraper.cpp

```diff
@@ -99,12 +98,30 @@ Skyscraper::Skyscraper(const QCommandLineParser &parser,
     printf("%s", StrTools::getVersionHeader().toStdString().c_str());
 
     config.currentDir = currentDir;
-    loadConfig(parser);
```

Comment:
Moved the `loadConfig` call to the main function because it contains `exit()` calls, which are not safe to invoke within a constructor. Calling `exit()` in a constructor can lead to memory leaks and other resource management issues.